README.md

```diff
@@ -104,6 +104,7 @@ To access the Wordpress Admin at `/wp-admin`, the default values are as follows:
 | `DB_HOST`          | `db`                             | Hostname for the database                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 | `DB_NAME`          | `wordpress`                      | Name of the database                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | `DB_PREFIX`        | `wp_`                            | Prefix for the database                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| `DB_CHARSET`        | `utf8`                            | Select a charset for the wordpress database (legacy versions might not be utf8)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
Can you make sure the pipes | are aligned, aka that there are no extra space between lines?